pdf-aam/index.html

```diff
@@ -226,81 +183,549 @@ <h4 id="role-map-example"><code>example</code></h4>
                   <abbr title="macOS Accessibility Protocol">AX API</abbr><sup>[<a href="#ftn.note1">Note 1</a>]</sup>
                 </th>
                 <td>
-                  <span class="property">AXRole: <code>AXxample</code></span
+                  <span class="property">AXRole: <code>Use WAI-ARIA mapping</code></span
+                  ><br />
+                  <span class="property">AXSubrole: <code>Use WAI-ARIA mapping</code></span
+                  ><br />
+                </td>
+              </tr>
+              <tr>
+                <th><abbr title="Notes">Notes:</abbr></th>
+                <td>
+                  <span class="property"
+                    >Note from PDFa:
+                    <code
+                      >Added note - question for the group... A screen reader shouldn't "announce" when it hits the first Document tag but should it "announce" when it hits "nested" (subsequent)
+                      Document tags, to tell someone they are in a "second" document? AND... the same question applies to Document Fragment, too. If "Yes," how do we indicate that in the AAM?</code
```

Comment:
Normative AT requirements are out of scope for the AAM docs, but you can put in informative hints. For example, AT might treat this like a separate document, such as they do with an iframe.